MediaContentSynchronization/Model/Consumer.php

```diff
@@ -12,7 +12,7 @@
 /**
  * Media content synchronization queue consumer.
  */
-class Consume
+class Consumer
```

Comment:
I wouldn't change the naming to keep the consistency. I.e. for Magento, it's more common to use action based name (`SaveAsset`) naming rather than object-based name (`AssetSaver`), especially when the only public method is `execute`. What do you think @lenaorobei @sidolov ?

Reply:
In this case we need to rename all Writers and Readers for consistency

Reply:
I feel like `Consumer` sounds more natural. Left as is.

Reply:
Considering that there are different opinions regarding the class naming, shouldn't we exclude this change from refactoring and leave the name as it is? Or is that an important change?

Reply:
Changed to `Consume` 🙂